zoneminder-latest.json

```
@@ -0,0 +1,80 @@
+                    "PUID": {
+                        "description": "Enter a valid UID to run zoneminder with. It must have full permissions to all Shares mapped in the previous step.",
+                        "label": "UID",
+                        "index": 1	
+                    },
+                    "PGID": {
+                        "description": "Enter a valid GID to use along with the same UID. It(or the above UID) must have full permissions to all Shares mapped in the previous step.",
+                        "label": "GID",
+                        "index": 2
+                    },
+                    "INSTALL_HOOK": {
+                        "description": "0=Disable or 1=Install hook processinging packages and run setup.py to prepare the hook processing",
+                        "label": "INSTALL_HOOK",
+                        "index": 3
+                    },
+                    "INSTALL_FACE": {
+                        "description": "0=Disable or 1=Install face recognition packages",
+                        "label": "INSTALL_FACE",
+                        "index": 4
+                    }
+                },
+                "ports": {
+                    "80": {
+                        "description": "Insecure port for running ZoneMinder. You shouldn't open this externally (it's unsecure). Do not use 80 (it will interfere with RockStor)",
+                        "host_default": 8080,
+                        "protocol": "tcp",
+                        "label": "ZoneMinder Insecure Port"
+                    },
+                    "443": {
+                        "description": "Secure port for running ZoneMinder. This will use a self-signed SSL certificate for now.",
+                        "host_default": 8443,
+                        "protocol": "tcp",
+                        "label": "ZoneMinder Secure Port",
+                        "ui": true
+                    },
+                    "9000": {
+                        "description": "zmninja monitoring port",
+                        "host_default": 9000,
+                        "protocol": "tcp",
+                        "label": "ZMNinja Port"
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for the config.",
+                        "label": "ZoneMinder Config",
+                        "min_size": "1073741824"
+                    },
+                    "/var/cache/zoneminder": {
+                        "description": "Choose a Share for the cache.",
+                        "label": "ZoneMinder Cache",
+                        "min_size": "1073741824"
+                    }
+                }
+            }
+        },
+        "description": "ZoneMinder: Free, open-source software to control IP, USB and Analog (CCTV) cameras (v1.32) - please note this runs as privileged in docker (to set shm to a higher amount)",
```

Add link to docker image:

```suggestion
        "description": "ZoneMinder: Free, open-source software to control IP, USB and Analog (CCTV) cameras (v1.32) - please note this runs as privileged in docker (to set shm to a higher amount).<p>Based on the following docker image: <a href='https://github.com/dlandon/zoneminder' target='_blank'>https://github.com/dlandon/zoneminder</a></p>.",
```